airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -646,13 +606,8 @@ def patch_task_instance(
     return task_instance_reference_schema.dump(ti)
 
 
-@security.requires_access(
-    [
-        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_DAG),
-        (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG_RUN),
-        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_TASK_INSTANCE),
-    ],
-)
+@security.requires_access_dag("GET", DagAccessEntity.RUN)
```

Comment:
Same here -> this call can potentially create dag_runs.

Reply:
I removed `@security.requires_access_dag("GET", DagAccessEntity.RUN)`